Assignments/Program.cs

```diff
@@ -1 +1,35 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T14 - Reduced String
+// ---------------------------------------------------------------------------------------
+
+using System;
+using static System.ConsoleColor;
+namespace Spark;
+
+public class T14 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "String Reducer:-" + "\x1B[0m");
+      Console.WriteLine ("Here, the input is a string, the output is a reduced string of lowercase characters by deleting a pair of adjacent letters that match. ");
+      string[] testCases = { "aaabccddd", "aaaabbbb", "AAbcXXY", "xyz1233345667", "  " };
+      Console.WriteLine ("\n\x1B[4m" + "Test Cases:-" + "\x1B[0m");

```

Comment:
Use Console.WriteLine instead of \n as \r\n is the line ending in Windows and in Linus it is \n.